src/dialogs/PaymentErrorDialog.vue

```diff
@@ -29,8 +29,8 @@
         <div class="font-15 mb-4" v-if="numErrors > 0">
           <p>We were unable to process your payment due to the following errors:</p>
           <ul>
-            <li v-for="(error, index) in errors" :key="index">
-              {{error.message}}
+            <li v-for="(error, index) in errors[0].message.split('<br/>')" :key="index">
+              {{ error }}
```

Comment:
This part the fix for [9014](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/9014): HTML tags show up in the warning popup window.

Reply:
This change will show each line as a separate bullet.

I think the intent is to show a formatted line... you could use the `v-html` directive for that. I'm not sure if the lines will all align correctly so this needs to be tested.

Reply:
^^ Check with UI/IX on both options.

Reply:
yeah, I think we should be using `v-html` for this.  There could be potentially other types of html tags coming back from pay api too.

Reply:
![image](https://user-images.githubusercontent.com/51458187/164786557-d9c9a53c-8596-47aa-84ba-917480aa6916.png)

Reply:
I couldn't find the proper UI/UX for this page, and I wasn't able to test this on my local environment.

Reply:
I think `v-html` looks ok.  The only thing is I would probably add styling bullet text indents after the first line.

i.e.
![image](https://user-images.githubusercontent.com/6685223/164787688-48987fcb-cdcb-4cde-bdaa-51aa62363d6a.png)

Reply:
![image](https://user-images.githubusercontent.com/51458187/164789020-5548d064-6908-4291-9a34-1daa7179270d.png)

If we were to use v-html, then the backend needs to send back the error string in this way specifically for the frontend to render the v-html with .error-span class (or some other name).

Reply:
The current message is the following:
"An error occurred during the BC Online transaction.<br/>Please contact the help desk to resolve this issue.<br/>SERVCE BC HELP DESK:<br/>Toll-free:1800-663-6102 (Canada and USA only)<br/>Email:bcolhelp@gov.bc.ca."

The customized message to render the v-html above is like this:
'An error occurred during the BC Online transaction.<br/><span class="error-span">Please contact the help desk to resolve this issue.</span><br/><span class="error-span">SERVCE BC HELP DESK:</span><br/><span class="error-span">Toll-free:1800-663-6102 (Canada and USA only)</span><br/><span class="error-span">Email:bcolhelp@gov.bc.ca.</span>'

Reply:
We do not want to change the BE to wrap each line in a span with a class.

You're trying to use the span to get the alignment correct, right? There's probably another way that works with your latest example message.

Reply:
Worse case, you could change each list item to a flex box and use shrink/grow so the bullet is in one "column" and the text is in another.

Example (left side of this diff): https://github.com/bcgov/business-filings-ui/pull/315/commits/afa6033cdde3e7e5e0ed0ca0c3a48167278487bf

Reply:
@Lcyc29 As there is no design doc for this, ask Yui or Tracey how they would like to see this.

Reply:
@argush3 Thanks Argus for the feedback. Please keep giving me feedback :)

Reply:
![image](https://user-images.githubusercontent.com/51458187/164801446-ea075ba0-62d7-49bc-b8a3-1141cec0aa00.png)

I used a local variable to transform the error message into tags that can use css styles. It looks okay, but I am unable to test this in my local environment and the unit test returns an error showing that there is no <li> tag element.